src/modules/pages/NewApiPage/NewApiForm.js

```diff
@@ -19,62 +31,272 @@ const ApiForm = (props) => {
-                <Section>
-                    <FormRow>
-                        <FormLabel text="Health check" />
-                    </FormRow>
-                    <FormRow>
-                        <FormInput component="input" label="url" attachTo="health_check.url" type="text" />
-                        <FormInput component="input" label="timeout" attachTo="health_check.timeout" type="text" parse={parse} />
-                    </FormRow>
-                </Section>
+                <Row>
+                    <Title>Create New API</Title>
+                    <Button
+                        type="submit"
+                        mod="primary"
+                    >
+                        Create API
+                    </Button>
+                </Row>
             </Section>
+            <div className={b('inner')}>
+                <div className={b('section')}>
+                    <div className={b('section-title')}>1. General</div>
+                    <Row className={b('row')()} fullwidth>
+                        <Row col>
+                            <Label>API Name</Label>
+                            <Field
+                                name="name"
+                                type="text"
+                                component={Input}
+                            />
+                            <Hint>Must be unique</Hint>
+                        </Row>
+                        <Row col>
+                            <Label>Is Active?</Label>
+                            <Row className={b('radio-wrap')()}>
+                                <Row className={b('radio')()}>
+                                    <Field
+                                        name="active"
+                                        component={Radio}
+                                        value={'true'}
+                                        type="radio"
+                                        id="is-active"
+                                    />
+                                    <Label htmlFor="is-active">Yes</Label>
+                                </Row>
+                                <Row className={b('radio')()}>
+                                    <Field
+                                        name="active"
+                                        component={Radio}
+                                        value={'false'}
+                                        type="radio"
+                                        id="is-not-active"
+                                    />
+                                    <Label htmlFor="is-not-active">No</Label>
+                                </Row>
+                            </Row>
+                        </Row>
+                    </Row>
+                </div>
+                <div className={b('section')}>
+                    <div className={b('section-title')}>2. Proxy</div>
+                    <Row className={b('row')()} fullwidth>
+                        <Row col>
```

Comment:
Same here this rows could also be taken into a component

Reply:
in separate **PR** I will remove all these layout components in favor of `classNames`.